README.md

```diff
@@ -11,47 +15,66 @@ This project was inspired by Arsunt's
+
+1. Head over to GitHub releases: https://github.com/rr-/Tomb1Main/releases
+2. Download the installer. Your browser may complain that the .exe is unsafe, but it's OK to ignore this alert.
+3. Mark the installer EXE as safe to run by right-clicking on the .exe, going to properties and clicking "Unblock".
+4. Run the installer and proceed with the steps.
+
+We hope that eventually these alerts will go away as the popularity of the project rises.
+
+## Installing (advanced / manual)
+
+1. Head over to GitHub releases: https://github.com/rr-/Tomb1Main/releases
+2. Download the zip file.
+3. Extract the zip file into a directory of your choice.  
    Make sure you choose to overwrite existing directories and files
-   (`Tomb1Main_config.json5` can remain).
-
-3. (**Steam users / music**) The Steam version of the game does not ship with the
-   music assets due to the publishers' negligence. If you have previously
-   installed TombATI, you don't need to do anything. Otherwise you can download
-   the music files from the link below.  
-   https://mega.nz/file/f9llhQAY#y0RqaMhR4ghtQ-1IFAGbHep_FCmkV8Q66bzdMWVqtuY  
-   **These assets are not a part of this project.** The legality of these is
-   disputable. The most legal way to get the music on PC is to rip the assets
-   yourself from a physical PlayStation disk. TombATI ships with this music by
-   default, which this project chooses not to do.
-
-4. Launch `Tomb1Main.exe`.
-
-### Unfinished Business
-
-1. (**Steam users / Unfinished Business**) The Unfinished Business expansion
-   pack is freeware, however, the Steam version of the game does not ship with
-   it and you'll need to install it separately. If you have previously
-   installed TombATI, you don't need to do anything. Otherwise you can download
-   the expansion pack from the link below.  
-   https://mega.nz/file/i8s0BAaD#wy3xyhuaHnco7aJKiKUV3zO6vCe86lLa3W7hWu249-M
-
-2. Launch the game with `-gold` command line switch. A more detailed instruction:
-
-    1. Create a shortcut to `Tomb1Main.exe`.
-
-    2. Select the newly created shortcut, go to File Properties.
-
-    3. At the end of the "target" field, append `-gold` so it looks something like
-        this:  
-        `(...)\Tomb1Main.exe -gold`
-
-    4. Launch the shortcut.
+   (`cfg/Tomb1Main_config.json5` can remain).
+4. (First time installation) Put your original game files into the target directory.
+    1. For Steam and GOG users, extract the original `GAME.BIN` file using a tool such as UltraISO to your target directory.
+       Note that neither the GOG nor the Steam releases ship the music files. You have a few options here:
+       - You can download the music files from the link below.  
+         https://tmp.sakuya.pl/tomb1main/music.zip
+         The legality of this approach is disputable.
+       - Rip the assets yourself from a physical PlayStation/SegaSaturn disk.
+       Optionally you can also install the Unfinished Business expansion pack files.
```

Comment:
This line should receive its own bullet point, or be somehow linked with the two UB points following. It's currently part of the instruction on audio ripping.

Reply:
Fixed